codegen/lib/graphql_java_gen.rb

```diff
@@ -274,4 +274,34 @@ def abstract_types
       end
     end
   end
+
+  def java_doc(element)
+    doc = ''
+    unless element.description.nil?
+      description = wrap_text(element.description, 100)
```

Comment:
If this is a concern for Shopify's schema descriptions, then can't we just add the appropriate wrapping there?